projects/angular/src/forms/combobox/_combobox.clarity.scss

```diff
@@ -190,129 +101,55 @@
-        variables.$clr-use-custom-properties
-      );
-      border-width: variables.$clr_baselineRem_1px;
+      @include mixins.generate-typography-token(
+        'CAPTION-LG-11',
+        (
+          font-weight: combobox-variables.$clr-combobox-pill-font-weight,
+          font-size: combobox-variables.$clr-combobox-pill-font-size,
+          line-height: combobox-variables.$clr-combobox-pill-line-height,
+          letter-spacing: combobox-variables.$clr-combobox-pill-letter-spacing,
+        )
+      );
+      height: combobox-variables.$clr-combobox-pill-height;
+      margin: combobox-variables.$clr-combobox-pill-margin;
+      background-color: combobox-variables.$clr-combobox-pill-background-color;
+      border-width: tokens.$cds-global-space-1;
       border-style: solid;
+      align-items: center;
 
-      // ATM Remove fallback and property
-      @include mixins.css-var(
-        align-items,
-        clr-combobox-pill-align-items,
-        baseline,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        border-radius,
-        clr-combobox-pill-border-radius,
-        combobox-variables.$clr-combobox-pill-border-radius,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        border-color,
-        clr-combobox-pill-border-color,
-        combobox-variables.$clr-combobox-pill-border-color,
-        variables.$clr-use-custom-properties
-      );
+      border-radius: combobox-variables.$clr-combobox-pill-border-radius;
+      border-color: combobox-variables.$clr-combobox-pill-border-color;
 
-      @include mixins.css-var(
-        padding,
-        clr-combobox-pill-padding,
-        0 variables.$clr_baselineRem_2px 0 variables.$clr_baselineRem_4px,
-        variables.$clr-use-custom-properties
-      );
+      padding: combobox-variables.$clr-combobox-pill-padding;
 
       .clr-combobox-pill-content {
-        @include mixins.css-var(
-          color,
-          clr-combobox-pill-font-color,
-          combobox-variables.$clr-combobox-pill-font-color,
-          variables.$clr-use-custom-properties
-        );
-        font-size: variables.$clr-typography-xxsmalltext;
-        font-weight: variables.$clr-font-weight-regular;
-        @include mixins.css-var(
-          padding,
-          clr-combobox-pill-content-padding,
-          0 variables.$clr_baselineRem_8px 0 variables.$clr_baselineRem_4px,
-          variables.$clr-use-custom-properties
-        );
+        font-size: tokens.$cds-alias-typography-caption-font-size;
+        font-weight: tokens.$cds-alias-typography-caption-font-weight;
+        color: combobox-variables.$clr-combobox-pill-font-color;
+        padding: 0;
       }
 
       cds-icon,
       clr-icon {
-        @include mixins.css-var(margin-top, clr-combobox-item-icon-override, 0, variables.$clr-use-custom-properties);
+        margin-top: calc(-1 * #{tokens.$cds-global-space-1});
```

Comment:
For refactoring ticket `margin-top`: -1px` should be done at `.clr-combobox-pill-content`. Also we need design for icons in pills, not only for close icon.

Reply:
Please, let's keep to the scope of the current task.
The clr-icon may **not** have a margin-top at all after redesigning the DOM.